README.md

```diff
@@ -0,0 +1,86 @@
+# auxtools
+
+auxtools is a Rust-based successor to the [Extools](https://github.com/MCHSL/extools) project.
+
+Currently, it implements some of the features such as the debug server, but is planned to supplant most functionality (maptick, etc.).
+
+## Features
+Debug Server - Working with SpaceManiac's [SpacemanDMM](https://github.com/SpaceManiac/SpacemanDMM), Auxtools interfaces with the debugger frontend to provide debugger information and management of breakpoints.
+
+*Dissassembly*  - When currently stopped at a breakpoint, enter the `#dis` command into the Debug Console in VSC to see the DM bytecode for the current proc. You can also provide it any arbitrary proc path like `#dis /mob/proc/Life`.
+
+## Dependencies
+
+*These instructions were taken directly from tgstation's [rust-g](https://github.com/tgstation/rust-g) documentation.*
+
+The [Rust] compiler:
+
+1. Install the Rust compiler's dependencies (primarily the system linker):
+
+   * Ubuntu: `sudo apt-get install gcc-multilib`
+   * Windows (MSVC): [Build Tools for Visual Studio 2017][msvc]
+   * Windows (GNU): No action required
```

Comment:
Windows GNU builds do work, but they're expected to crash and burn horribly in some situations so it'd be best to specify MSVC is required.